src/rpc/quorums.cpp

```diff
@@ -136,6 +140,21 @@ static UniValue quorum_list_extended(const JSONRPCRequest& request)
 
         auto quorums = llmq_ctx.qman->ScanQuorums(type, pblockindex, llmq_params.signingActiveQuorumCount);
         for (const auto& q : quorums) {
+            size_t num_members = q->members.size();
+            size_t num_valid_members = 0;
+            for (size_t i = 0; i < num_members; i++) {
+                if (q->qc->validMembers[i]) {
+                    num_valid_members++;
+                }
+            }
```

Comment:
we should be able to use std::count here right?

Reply:
@PastaPastaPasta If the quorum was created with `minSize` members, then using `std::count` will lead to wrong return value.
Because `q->qc->validMembers[]` has always the same size with the expected quorum size which is greater than the `num_members` in this example.

Reply:
I don't see why you can't still use std::count I believe that validMembers values are initialized to false, so std::count over the whole range should work right? If not we could still do something like
```
            size_t num_valid_members = std::count_if(q->qc->validMembers.begin(), q->qc->validMembers.begin() + num_members, [](auto val){return val;});
```

Reply:
@PastaPastaPasta Yup will do that

Reply:
```
            size_t num_members = q->members.size();
            size_t num_valid_members = std::count_if(q->qc->validMembers.begin(), q->qc->validMembers.begin() + num_members, [](auto val){return val;});
            double health_ratio = num_members > 0 ? double(num_valid_members) / double(num_members) : 0.0;
            std::stringstream ss;
            ss << std::fixed << std::setprecision(2) << health_ratio;
            UniValue obj(UniValue::VOBJ);
```